docs/learn/learn-async-backing.md

```diff
@@ -7,104 +7,312 @@ keywords: [parachains, backing, parablock, perspective parachains, unincluded se
+P4).
 
-A particular parablock, `P1`, would **only** be valid for backing at relay chain parent `R1 + 1` and
-subsequently, be included at `R1 + 2` should it be backed successfully. Essentially, a parablock is
-rushing to be backed and included within this two-block window due to the inherent requirement for
-synchrony between the parachain and relay chain.
+In this scenario, blockspace demand decreases from P3 to P5. In theory, the next block P6 can be
+filled in >2s seconds while P4 is included and P5 is backed. Collators could make better use of
+blockspace, including more data by increasing block generation time in a period of lower demand.
+Note that even if a collator has >2 seconds to produce a block, the validator will still have less
+than 6 seconds (~2 seconds) to check it. So, if collators take >2 seconds to generate blocks the
+unincluded segment will shrink (and there will be less parablocks in the pipeline), while if they
+take <2 seconds the segment will elongate until it reaches the `maximum capacity` (and there will be
+less parablocks in the pipeline).
 
-## Asynchronous Backing on Polkadot
+For example, in case increased blockspace demand after the generation of P6, collators could
+generate P7 and P8 within a 6-second window due to the unincluded segment being 2/3 filled (see
+below). However, if demand stays high after P8, collators will only be able to generate parablocks
+every 6 seconds as validators can back parablocks every 6 seconds.
 
-With asynchronous backing, the window of time is customizable and will most likely sit around the
-6-18 second range. It also introduces a parameter to aid in defining the maximum amount of ancestor
-blocks, which allows for a parablock to be backed later in the future, enabling more computational
-and storage time per block. The context for the latest parablock is derived from the unincluded
-segment of block ancestors upon which the newest parablock is built.
+![async-backing-fill-scenario2](../assets/async-backing-fill-scenario2.png)
 
-These blocks can be prepared in anticipation of being included later rather than keeping in sync
-with the relay chain's progress 1-1. The block can be in a different stage from another block being
-built, as long as it abides by the parameters set forth by the asynchronous backing configuration.
+In general, asynchronous backing will allow [collators](./learn-parachains-protocol.md#collators) to
+include an estimated ~3-5x more data into parablocks while speeding up parachain block times from 12
+to 6 seconds. Due to the 2x decrease in block time to inclusion and the possibility of building
+blocks in advance to fit more data, Polkadot with asynchronous backing can deliver an estimated
+~6-10x more blockspace to its parachains.
 
 This combination of lower latency, higher storage per block, and a logical pipeline spanning
 Polkadot's networking, runtime, and collation aspects will allow for higher, more robust throughput.
```

Comment:
This doesn't increase storage per block, it increase compute time for collators (what we call "authoring duration" in Cumulus).

"Logical pipeline spanning Polkadot's networking, runtime, and collation aspects" doesn't make sense. Pipelining means you build blocks in advance before they're included. It has little to do with networking or the Polkadot runtime. 

"More rubust throughput" doesn't make sense.

Reply:
deleted